[OSDOCS-15022]UPDATE Prereq order in OSD docs/add WIF recommendation in Note (#95164)
openshift/openshift-docs: modules/ccs-gcp-customer-procedure.adoc

> The Customer Cloud Subscription (CCS) model allows Red{nbsp}Hat to deploy and manage {product-title} into a customer's Google Cloud Platform (GCP) project. Red Hat requires several prerequisites to provide these services.

Reviewer: ```suggestion
The Customer Cloud Subscription (CCS) model allows Red{nbsp}Hat to deploy and manage {product-title} into a customer's Google Cloud Platform (GCP) project. Red{nbsp}Hat requires several prerequisites to be completed before providing these services.
```

Know this is outside scope for this PR but this sentence doesn't make sense as is. Struggling with the wording a bit on this one - this is the best I could come up with.

[NOTE]
====

> The following requirements in this topic apply to {product-title} on {GCP} clusters created using both the Workload Identity Federation (WIF) and service account authentication type.

Reviewer: ```suggestion
The following requirements in this topic apply to {product-title} on {GCP} clusters created using both the Workload Identity Federation (WIF) and service account authentication types.
```

> Red{nbsp}Hat recommends using WIF as the authentication type for installing and interacting with the OpenShift Dedicated cluster deployed on GCP because it provides enhanced security.

Reviewer: ```suggestion
Red{nbsp}Hat recommends using WIF as the authentication type for installing and interacting with an {product-title} cluster deployed on GCP because WIF provides enhanced security.
```
